packages/shared/components/inputs/Pin.svelte

```diff
@@ -133,10 +134,9 @@
 <div class="w-full {classes}">
     <pin-input
         style="--pin-input-size: {PIN_LENGTH}"
-        class="{$mobile ? 'h-20 w-80' : 'w-full'}
-            flex items-center justify-between relative z-0 rounded-xl border border-solid
+        class={`flex items-center justify-between w-full relative z-0 rounded-xl border border-solid
             bg-gray-50 dark:bg-gray-800 border-gray-300 dark:border-gray-700
-            {smaller ? 'h-14 pl-6 pr-4' : $mobile ? 'h-20 pl-8 pr-8 m-auto' : 'h-20 pl-12 pr-8'}"
+            ${smaller ? 'h-14 pl-6 pr-4' : 'h-20 pl-12 pr-8'}`}
```

Comment:
@amadeu2 Have you already changed the login PIN regarding the new UI design somehow? Perhaps we should add w-5/6 instead of w-full for mobile and get rid of the delete button like in the new UI Layout (figma -> login).

Reply:
Yes should need to be adapted to the current changes on `develop`. And we should test it on every screen size as the current one before merge. 
Also considering what Begonia said today about to merge the less PRs as possible before the 1.5 merge, we could await a bit and merge it before the end of the week, and test it with time.